src/HIVCohortFilter/HIVCohortCases.jsx

```diff
@@ -0,0 +1,1205 @@
+import React from 'react';
+import FileSaver from 'file-saver';
+import Button from '@gen3/ui-component/dist/components/Button';
+import { arrangerGraphqlPath } from '../localconf';
+import { fetchWithCreds } from '../actions';
+import './HIVCohortFilter.css';
+import CohortPTCSvg from '../img/cohort-PTC.svg';
+import CohortECSvg from '../img/cohort-EC.svg';
+import CohortLTNPSvg from '../img/cohort-LTNP.svg';
+import Spinner from '../components/Spinner';
+
+class HIVCohortFilterCase extends React.Component {
+  // Base class for the 3 NDH cohort filter apps. Meant to facilitate code reuse
+  constructor(props) {
+    super(props);
+    this.updateFilters = this.updateFilters.bind(this);
+    this.checkReadyToCalculate = this.checkReadyToCalculate.bind(this);
+    this.downloadControl = this.downloadControl.bind(this);
+    this.therapyValuesOfInterest = ['HAART'];
+  }
```

Comment:
there is no default state but I see you adding state later - should add a default state, and maybe `therapyValuesOfInterest` can be in it ?

Reply:
✓